test/integration/test_httproute.go

```diff
@@ -190,3 +193,204 @@ func TestHTTPRouteV1Beta1(t *testing.T) {
 		httpRouteAccessTimeout, time.Second,
 	)
 }
+
+func TestHTTPRouteWithTLS(t *testing.T) {
+	t.Parallel()
+	namespace, cleaner := helpers.SetupTestEnv(t, GetCtx(), GetEnv())
+
+	t.Log("deploying a GatewayConfiguration to set KIC log level")
```

Comment:
```suggestion
```
With `t.Logf("deploying GatewayConfiguration %s/%s to set KIC log level", gatewayConfig.Namespace, gatewayConfig.Name)` below I think this log is redundant.